src/TextareaAutosize.js

```diff
@@ -87,6 +93,13 @@ export default class TextareaAutosize extends React.Component {
     // Re-render with the new content then recalculate the height as required.
     this.onNextFrameActionId = onNextFrame(this._resizeComponent);
   }
+  
+  componentWillUpdate(nextProps, nextState) {
```

Comment:
@chrislloyd Probably invoke callback from *componentDidUpdate*?
@andreypopp yes, it can lead into endless loop. Do you have any idea, how to prevent this?